### PR TITLE
Add slack button in header

### PR DESCRIPTION
**Pull request by samscott89**

### CODE REVIEW

File: docs/_templates/header.html [hunk 1]
```diff
@@ -27,14 +27,28 @@
             {% if theme_repo_url %}
             <div class="md-flex__cell md-flex__cell--shrink">
                 <div class="md-header-nav__source">
-                    <!-- {% include "repo.html" %} -->
+                    <a href="https://oso-oss.slack.com/">
```
Comment (gj) [inline docs/_templates/header.html:30]: This should be the invite link

Comment (samscott89) [reply docs/_templates/header.html:30]: Good point!

**Comment by samscott89** [comment]
> do you have screenshots of the difference?

image

**Comment by samscott89** [comment]
Sorry @gj , tacked a few other changes on. Updated the button font to the same as osohq.com:

image

**Comment by gj** [comment]
@samscott89 it looks like there's a faint black border around those buttons -- it doesn't look good to my eye

**Pull request merged**